jipdate.py

```diff
@@ -274,6 +278,9 @@ def parse_status_file(jira, filename):
     # [FIN]
     regex_fin = r"^\[FIN\]\n$"
 
+    # Regexp to match for a status update:
+    regex_status = r"^Status: (.+)\n$"
```

Comment:
I think this deserves a short explanation in the README.md file also explaining how to do a transition. I.e., one need to remove the comment and write the transition name exactly as in Jira. Btw, all CAPS doesn't work even though that can be seen in Jira when looking at the workflow. So maybe we should automatically re-format the string so the transition name becomes "Word Capitalized" no matter how you write it (this seems to be what Jira expects).